Items/04_01_Melee_&_Thrown_Weapons.txt

```diff
@@ -21,26 +21,45 @@ Damage: 30 + STR mod + DEX mod Ballistic, minimum 10
 Range: Adjacent (1m)
 Accuracy Modifier: 0
 
+	Ranged Miss Chance
+
+* 2 out to 5m
+* 5 out to 20m
+
 	Some improvised weapons may have alternate damage types, at the DM's 
-discretion.
+discretion. Note that the miss chance stat is only relevant when throwing an 
+improvised weapon, while the accuracy is only used for melee attacks.

```

Comment:
great clarification! I love how I can understand these rules well the first time I read them. That wasn't always true with CX rules.